documentation/modules/running-migration-plan.adoc

```diff
@@ -21,19 +21,40 @@ The *Plans* list displays the source and target providers, the number of virtual
 . Click *Start* beside a migration plan to start the migration.
 . Click *Start* in the confirmation window that opens.
 +
-The *Migration details by VM* screen opens, displaying the migration's progress
+The plan's *Status* changes to *Running* and migration's progress is displayed.
 +
 Warm migration only:
 
 * The precopy stage starts.
 * Click *Cutover* to complete the migration.
 
-. If the migration fails:
-.. Click *Get logs* to retrieve the migration logs.
-.. Click *Get logs* in the confirmation window that opens.
-.. Wait until *Get logs*  changes to *Download logs* and then click the button to download the logs.
+. Click the links in the migration's *Status* to see its overall status and the status of each VM:
 
-. Click a migration's *Status*, whether it failed or succeeded or is still ongoing, to view the details of the migration.
+** The link on the left indicates whether the migration failed, succeeded, or is ongoing and reports the number of VMs whose migration succeeded, failed, or was canceled.
+** The link on the right opens the *Virtual Machines* tab of the *Plan Details* page. For each VM, the tab displays the name of the VM, the start and end times of the migration, the amount of data copied, and a progress pipeline for its migration.
```

Comment:
I'd maybe suggest to turn this into a bulleted list, but it's not necessary, so feel free to ignore :-) If you'd like to use the list, I'd suggest something like:

> For each VM, the tab displays the following data:
> +
> *** The name of the VM
> *** The start and end times of the migration
> *** The amount of data copied
> *** A progress pipeline for the VM's migration
> 
> 
> 
>

Reply:
Good idea.